app/views/content/train-to-be-a-teacher/teacher-degree-apprenticeships.md

```diff
@@ -21,19 +21,30 @@ If you do not already have an undergraduate degree, you may be eligible to do a
 
 $teacher-degree-apprenticeships$
 
-##  How do teacher degree apprenticeships (TDAs) work?
-This new teacher training programme means you’ll work in a school and earn a salary while getting a bachelor’s degree and [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts).
+##  How do teacher degree apprenticeships work?
+The teacher degree apprenticeship is a new teacher training programme where you’ll work in a school and earn a salary while getting a bachelor’s degree and [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts).
  
 The teaching apprenticeship is a 4-year, full-time course that lets you train as either a primary or secondary teacher with an accredited teacher training provider. Trainee teachers will spend an average of 40% of their time (two days a week) studying for their degree with a university.
 
-The salary you’ll earn as a trainee will reflect the level of responsibility that’s suitable for each stage of your course. You do not have to pay tuition fees but you will not be eligible for student finance.
+The salary you’ll earn as a trainee will reflect the level of responsibility that’s suitable for each stage of your teaching apprenticeship course. You do not have to pay tuition fees but you will not be eligible for student finance.
 
 The [teacher training providers confirmed so far](https://www.gov.uk/government/publications/secondary-maths-teacher-degree-apprenticeship-funding-pilot-providers?) will offer TDAs in maths, with other subjects likely to be offered too.
```

Comment:
@gemmadallmandfe tempted to remove the link to avoid confusion - but do you reckon it would be useful to users? Doesn't give a lot of extra info tbh!

Reply:
Agree - the gov.uk information is not aimed at candidates